### PR TITLE
Use casacore for conversion to ITRF

### PR DESCRIPTION
**Pull request by tammojan**
This would drop the dependence on pyproj

### CODE REVIEW

File: tools/create_casacore_table.py [hunk 1]
```diff
@@ -78,11 +77,13 @@
...
-    x, y, z = pyproj.transform(lla, ecef, row['longitude'],
-                               row['latitude'], row['elevation'])
+    [[x, y, z ]] =  tables.taql("calc meas.position('itrf',{}deg,{}deg,{}m,"
+                                "'wgsll')".format(row['longitude'],
+                                                  row['latitude'],
+                                                  row['elevation']))
     tablerows.put(i, {
         'MJD': 0.0,
-        'Name': row['Id'],
+        'Name': str(row['Id']),
```
Comment (olebole) [inline tools/create_casacore_table.py]: `row['Id']` should already be a `str`; why the assitional `str()`?

Comment (tammojan) [inline tools/create_casacore_table.py]: No idea why this `str` is necessary, but without this my python (2.7.10) threw this error:
```
Traceback (most recent call last):
  File "tools/create_casacore_table.py", line 91, in <module>
    'Source': row.get('Reference', 'Wikipedia')
  File "/Users/dijkema/opt/python-casacore/src/casacore/tables/tablerow.py", line 63, in put
    self._put(rownr, value, matchingfields)
RuntimeError: PycValueHolder: unknown python data type
```

Comment (olebole) [inline tools/create_casacore_table.py]: Ah, OK. WIth Python 2 `json.load()` creates `unicode` strings, and I tested only with Python 3 (where `str` is unicode).

**Comment by olebole** [comment]
Just to be sure: Can you confirm that the `X`, `Y`, and `Z` columns here really ITRF coordinates (even when the `Type` column is set to `"WGS84"`? I thought that the `Type` column specifies the type of the whole row, and so `X`, `Y`, and `Z` would contain the same system, as `Long`, `Lat`, and `Height`, just in cartesian coordinates.

**Comment by tammojan** [comment]
No sorry I cannot confirm that. I can check with higher authorities, if I can find them.

**Comment by tammojan** [comment]
I'm not sure why one would need both XYZ and Lat Long in one table, but probably there are historical reasons.

**Comment by olebole** [comment]
Yep. We don't need to set x,y,z hee (they are missing in the original table for many entries as well); however since it is simple, we could do it as a nice service ;-) At least if we know what exactly shall be there... If nobody knows, we can just drop this for the moment. What do you think?

**Comment by tammojan** [comment]
Thinking of it, the XYZ should probably (of course) refer to the same reference frame as the lat-long coordinates.  In the taql-snippet I suggested, replacing `itrf` with `wgs` should do the trick of converting to WGS X, Y, Z instead.

However, I don't think we're doing anyone (and in particular ourselves) a favor with creating duplicate information which will at some point diverge. Couldn't we just leave out the XYZ columns?

**Comment by olebole** [comment]
As long as the information is consistent, I don't see a problem here; space is not an issue for `Observatories`.
However I will do whatever you specialists recommend here. Can casacore handle an `Observatories` table without these columns? Would that need an API change?

**Comment by olebole** [comment]
The conversion looks a bit weird. A simple example:
```
TaQL> calc meas.position('wgs',0deg,0deg,0m,'wgsll')
using style python calc meas.position('wgs',0deg,0deg,0m,'wgsll')
    has been executed
Unit: m
[1e-06, 0, 0]
TaQL> calc meas.position('wgs',90deg,0deg,0m,'wgsll')
using style python calc meas.position('wgs',90deg,0deg,0m,'wgsll')
    has been executed
Unit: m
[6.12323e-23, 1e-06, 0]
TaQL> calc meas.position('itrf',0deg,0deg,0m,'wgsll')
using style python calc meas.position('itrf',0deg,0deg,0m,'wgsll')
    has been executed
Unit: m
[6.37814e+06, 0, 0]
TaQL> calc meas.position('itrf',90deg,0deg,0m,'wgsll')
using style python calc meas.position('itrf',90deg,0deg,0m,'wgsll')
    has been executed
Unit: m
[3.90548e-10, 6.37814e+06, 0]
```
While the `'itrf'` results look reasonable, the values for `'wgs'` are close to zero here? Somehow the earth radius is missing, or I didn't understand the coordinate system.

**Comment by olebole** [comment]
OK, I have no idea what the `'wgs'` (`xyz`) system here is, but I compared the `'itrf'` (`xyz`) with the values from the `pyproj` package: The deviation is always <~5e-16 and probably just due to the mathematical accuracy. Therefore I am going to merge here for pragmatic reasons.
I am open to drop the `X`, `Y`and `Z` columns completely if there is an OK that this will not cause problems in casacore.

**Pull request merged**